spirv_glsl.hpp

```diff
@@ -244,6 +249,7 @@ class CompilerGLSL : public Compiler
 		bool flexible_member_array_supported = true;
 		bool explicit_struct_type = false;
 		bool use_initializer_list = false;
+		bool transpose_row_major_matrices = false;
```

Comment:
I think supports_row_major_matrices is a bit clearer, as transposition is an implementation detail.